feat(curriculum): add Artifact Provenance Auditor workshop (#68860)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-artifact-provenance-auditor/6a57c331ae94fa4631b6086f.md

---
id: 6a57c331ae94fa4631b6086f
title: Step 16
challengeType: 1
dashedName: step-16
---

# --description--

> Update the curator of the artifact with an `id` of `101` to `"Fran Sinclair"`, then log that artifact's curator `name`.

Reviewer: ```suggestion
Update the curator for artifact `101` to `"Fran Sinclair"`. Then, log that artifact's curator `name`.
```
